server_installation/topics/operator/keycloak-cr.adoc

```diff
@@ -0,0 +1,67 @@
+
+=== Keycloak Custom Resource
+
+The {project_operator} allows users to manage {project_name} clusters by using Keycloak Custom Resources:
+
+.`Keycloak` Custom Resource
+```yaml
+apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
+metadata:
+  name: example-keycloak
+  labels:
+    app: sso
+spec:
+  instances: 1
+  externalAccess:
+    enabled: True
+```
+
+The `Spec` contains three objects (two of them are listed the above and the external database has been covered in the other paragraphs of this manual):
```

Comment:
What are those tree objects? `instances`, `externalAccess` and `enabled` ? Not getting, how / why it refers to something listed above - can it be more specific?

Reply:
Changed to "properties". Those are CR properties, more technically - Go structs.